#! /bin/sh /usr/share/dpatch/dpatch-run
## makefile-am-headers.dpatch by  <blair@ctrlpclnx.nscl.msu.edu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Don't let make install all of those unneeded headers

@DPATCH@

--- epicstcl-2.0-001-old/Makefile.am	2009-03-17 11:06:48.000000000 -0400
+++ epicstcl-2.0-001-new/Makefile.am	2009-03-17 14:57:45.000000000 -0400
@@ -8,7 +8,7 @@
 libException_la_SOURCES = ErrnoException.cpp Exception.cpp \
 			  RangeError.cpp StreamIOError.cpp \
 			StateException.cpp
-include_HEADERS = ErrnoException.h Exception.h  StreamIOError.h \
+noinst_HEADERS = ErrnoException.h Exception.h  StreamIOError.h \
 	RangeError.h StateException.h                           \
 	 TCLApplication.h TCLCommandPackage.h TCLException.h \
 	TCLFileHandler.h TCLHashTable.h TCLHashTableItem.h \
@@ -20,7 +20,8 @@
 	TCLObjectProcessor.h TCLCompatibilityProcessor.h histotypes.h \
 	CTCLChannelCommand.h CTCLEpicsCommand.h \
 	CTCLEpicsPackage.h	\
-	CChannel.h CChannelVariable.h
+	CChannel.h CChannelVariable.h \
+	Asserts.h
 
 libException_la_LDFLAGS = $(THREADFLAGS) -version-info 2:0:0 \
 	$(OSLDFLAGS)
@@ -177,8 +178,6 @@
 unittests_LDFLAGS = $(THREADFLAGS)
 unittests_LDADD   =  $(CPPUNIT_LDFLAGS) -lException $(TCL_LDFLAGS)
 
-noinst_HEADERS = Asserts.h
-
 tracetests_SOURCES = TestRunner.cpp \
 			ccallbacktest.cpp \
 			tracevartest.cpp
